definitions/contested/json/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -5585,6 +5599,13 @@
     "UserRole": "caseworker-divorce-financialremedy-judiciary",
     "CRUD": "CRUD"
   },
+  {
+    "LiveFrom": "01/01/2019",
+    "CaseTypeID": "FinancialRemedyContested",
+    "CaseFieldID": "Contested_ConsentedApprovedOrders",
+    "UserRole": "caseworker-divorce-financialremedy-judiciary",
+    "CRUD": "CRUD"
```

Comment:
do these need to be CRUD permissions when COS is the only place adding to the collection?